packages/sdk/src/endpoints/AttributesEndpoint.ts

```diff
@@ -44,6 +48,25 @@ export class AttributesEndpoint extends Endpoint {
         return await this.get("/api/v2/Attributes/Valid", request);
     }
 
+    public async getOwnRepositoryAttributes(request?: GetOwnRepositoryAttributesRequest): Promise<ConnectorResponse<ConnectorAttributes>> {
+        return await this.get("/api/v2/Attributes/Own/Repository", request);
+    }
+    public async getOwnSharedIdentityAttributes(request?: GetOwnSharedIdentityAttributesRequest): Promise<ConnectorResponse<ConnectorAttributes>> {
+        return await this.get("/api/v2/Attributes/Own/Shared/Identity", request);
+    }
+    public async getPeerSharedIdentityAttributes(request?: GetPeerSharedIdentityAttributesRequest): Promise<ConnectorResponse<ConnectorAttributes>> {
+        return await this.get("/api/v2/Attributes/Peer/Shared/Identity", request);
+    }
+    public async getVersionsOfAttribute(attributeId: string): Promise<ConnectorResponse<ConnectorAttributes>> {
+        return await this.get(`/api/v2/Attributes/${attributeId}/Versions`);
+    }
+    public async getSharedVersionsOfRepositoryAttribute(
+        attributeId: string,
+        request: GetSharedVersionsOfRepositoryAttributeRequest = { onlyLatestVersions: false }
```

Comment:
The value of the equivalent runtime use case parameter defaults to `true`. Is there a reason why the connector api should override this?

I don't have any problems with doing so, just asking as this might be a mistake.